lib/runner/index.js

```diff
@@ -4,7 +4,35 @@ var _ = require('lodash'),
     extractRunnableItems = require('./extract-runnable-items').extractRunnableItems,
 
     DEFAULT_GLOBAL_TIMEOUT = 3 * 60 * 1000, // 3 minutes
-    Runner;
+    Runner,
+
+    /**
+     * Converts any non finite values and 0 or negative values to Infinity.
+     *
+     * @param {Number} ms
+     *
+     * @returns {Number}
+     */
+    sanitizeTimeout = function (ms) {
+        return (_.isFinite(ms) && ms > 0) ? ms : Infinity;
+    },
+
+    /**
+     * Sanitizes `timeout` `options` if any on the options object.
+     *
+     * @param {Object} [timeoutOptions]
+     *
+     * @returns {Object}
+     */
+    sanitizeTimeouts = function (timeoutOptions) {
+        if (!timeoutOptions) { return timeoutOptions; }
```

Comment:
Shouldn't this be `return;`?

Reply:
Couple of checks that help us here,

1. `timeoutOptions` always has a default
2. We do not rely on return values on `sanitizeTimeouts`, the mutating behavior is relied on

That said you are right about falsy values, but just that the code flow never goes through.